src/topology/vector_bundle.lean

```diff
@@ -387,3 +404,57 @@ lemma is_open_map_proj : is_open_map Z.proj :=
 topological_fiber_bundle_core.is_open_map_proj Z
 
 end topological_vector_bundle_core
+
+end
+
```

Comment:
can you open a new section here and add a docstring explaining what you are going to do in this section?

Reply:
I added a title, should I repeat the docs of `topological_vector_prebundle`?

Reply:
I think it's ok like that because the definition comes right after the opening of the section.